float32[] position
bool hasFoundAllFrontiers
bool hasReachedCurrentFrontier
bool hasReachedNavigationTarget
bool hasFinishedAnalyzingTarget
string[] systemStatus
medos_demo/Target[] detectedTargets